### ISSUE TITLE
Change id on templates

### ISSUE THREAD

**Issue by GaryRidgway** [uiowa/uiowa#4995]
These @todos references removing the id attribute or changing it to a class for the student profile card template. We can remove the @todo when this is resolved.

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/modules/custom/grad_custom/grad_student_profile/templates/node--student-profile--card.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/sites/admissions.uiowa.edu/modules/admissions_core/templates/node--scholarship--teaser.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/sites/admissions.uiowa.edu/modules/admissions_core/templates/node--transfer-tips--teaser.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/sites/admissions.uiowa.edu/modules/admissions_core/templates/views-view-unformatted--student-profiles.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/sites/brand.uiowa.edu/modules/brand_core/templates/node--lockup--full.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/sites/hr.uiowa.edu/modules/hr_core/templates/node--faq--teaser.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/sites/ighn.international.uiowa.edu/modules/ighn_core/templates/node--ighn-person--full.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/sites/ighn.international.uiowa.edu/modules/ighn_core/templates/node--ighn-person--teaser.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/sites/ighn.international.uiowa.edu/modules/ighn_core/templates/node.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/sites/uipress.uiowa.edu/modules/uipress_core/templates/node--book--teaser.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/bba45460a6924a278b748d0599fc8e81bc7ce3fd/docroot/modules/custom/studentlife_topics/templates/node--topic-collection--teaser.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/bba45460a6924a278b748d0599fc8e81bc7ce3fd/docroot/modules/custom/uiowa_area_of_study/templates/node--area-of-study--teaser.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/themes/custom/uids_base/templates/content/node--area-of-study--full.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/themes/custom/uids_base/templates/content/node--article--full.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/themes/custom/uids_base/templates/content/node--article--teaser.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/themes/custom/uids_base/templates/content/node--event--teaser.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/themes/custom/uids_base/templates/content/node--page--full.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/themes/custom/uids_base/templates/content/node--page--teaser.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/themes/custom/uids_base/templates/content/node--person--teaser.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/themes/custom/uids_base/templates/content/node--teaser.html.twig#L68-L70

https://github.com/uiowa/uiowa/blob/68325126adc6d9be0b338b9c74fb074442422eff/docroot/themes/custom/uids_base/templates/content/node.html.twig#L68-L70

**Comment by bspeare** [comment]
Are these templates still outputting IDs?  I just checked `/docroot/sites/uipress.uiowa.edu/modules/uipress_core/templates/node--book--teaser.html.twig` and I don't see an ID rendering when the teaser is used in the related reads section on https://uipress.prod.drupal.uiowa.edu/cracking-black-feminist-comedy-twentieth-and-twenty-first-century-united-states.

**Comment by GaryRidgway** [comment]
I have no idea, I am just categorizing them all together, doing little investigation on them all, thought bundling them together would allow for low stoppage and better focus

**Comment by bspeare** [comment]
Yes, @GaryRidgway that wasn't meant for you especially.   It's great that they are grouped here.   Just pointing out that I didn't see an ID on one of them.   I imagine we copied these templates and reused them without removing the original `@todo`?

**Comment by GaryRidgway** [comment]
It seems like it. I have a few more to add
